embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/storage/EmbraceStorageService.kt

```diff
@@ -52,6 +55,21 @@ internal class EmbraceStorageService(
         return filesDir.toList() + cacheDir.toList()
     }
 
+    /**
+     * Logs storage telemetry, including the current size utilized by Embrace.
+     */
+    override fun logStorageTelemetry() {
+        val availableStorage = storageAvailabilityChecker.getAvailableBytes()
+        val storageUsed = listFiles { _, _ -> true }
+            .filter { it.isFile }
+            .sumOf { it.length() }
```

Comment:
will this `sumOf` be a `Long`? We have to make sure it won't overflow, andI think an 32-bit integer could

Reply:
Yup, it's a Long! And then we convert it to a String so we don't have any overflow risk unless we find a phone with 4 Exbibytes of available storage 😅